packages/peregrine/lib/talons/CartPage/GiftOptions/useGiftOptions.js

```diff
@@ -61,6 +76,8 @@ const useGiftOptions = props => {
     );
 
     /**
+     * @ignore
```

Comment:
I had to add the @ignore tag otherwise JSDoc will render this content block.
As far as I know, there isn't a way to tell JSDocs which blocks to accept/deny.

Reply:
Does it ignore non block comments ie 

```
// what about this
// even if it happens
// to span multiple lines
function thingy() {}
```

Reply:
yes, that would be a good workaround